azure-pipelines-template.yml

```diff
@@ -41,6 +43,16 @@ jobs:
       displayName: Install Watchman
 
   - script: npm install
```

Comment:
Should be `npm install --build-from-source` to ensure the CI doesn't download a prebuilt binary

Reply:
ah good catch, thanks. added on master.